src/System.Security.Permissions/src/System/Drawing/Printing/PrintingPermissionAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Permissions;
+
+namespace System.Drawing.Printing
+{
+    [AttributeUsage(AttributeTargets.All, AllowMultiple = true)]
+    public sealed class PrintingPermissionAttribute : CodeAccessSecurityAttribute
+    {
+        public PrintingPermissionAttribute(SecurityAction action) : base(action) { }
+        public PrintingPermissionLevel Level { get; set; }
+        public override System.Security.IPermission CreatePermission() { throw null; }
```

Comment:
We are only throwing out of CAS implementations where we believe that code written for desktop would be relying on CAS for some security purpose even when CAS is disabled. I think this was the case for one or two methods. Can be ignored for these types I think.